docs/src/main/asciidoc/grpc-service-implementation.adoc

```diff
@@ -328,6 +328,21 @@ public class HelloServiceTest implements Greeter {
 }
 ----
 
+== Trying out your services manually
+In the dev mode, you can try out your gRPC services in the Quarkus Dev UI.
+Just go to http://localhost:8080/q/dev and click on Services under the gRPC tile.
+
+Please note that your application needs to expose the "normal" HTTP port for the Dev UI to be accessible. If your application does not expose any HTTP endpoints, you can expose the port by adding the following dependency to your application:
+[source,xml]
+----
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx-http</artifactId>
+        </dependency>
+----
+
+NOTE: If you expose HTTP traffic only to access the Dev UI, consider adding this dependency in a dedicated profile and enabling it only when running `mvn quarkus:dev`
```

Comment:
It might make sense to mention that for Gradle, we can do something like:

```
dependencies {
    quarkusDev 'io.quarkus:quarkus-vertx-http'
}
```

Reply:
I didn't know this one, do I understand correctly that it adds a dependency to the application if `quarkusDev` task is executed?

Reply:
Exactly

Reply:
I added this info, and an example of a maven profile

Reply:
Great, thanks!